packages/contracts/test/test-utils/ens.ts

```diff
@@ -1,5 +1,6 @@
 import {ethers} from 'hardhat';
 import {SignerWithAddress} from '@nomiclabs/hardhat-ethers/signers';
+import { deployWithProxy } from './proxy';
```

Comment:
adjust the formatting to the other imports

Reply:
https://github.com/aragon/core/pull/154/commits/f636ab807bfb30abbc1c0f33ec0a2a52190f02f0